spec/collator.html

```diff
@@ -15,28 +15,31 @@
       </p>
 
       <p>
-        Several steps in the algorithm use values from the following table, which associates Unicode locale extension keys, property names, types, and allowable values:
+        Several steps in the algorithm use values from the following table, which associates Unicode locale extension keys, internal slots, property names, types, and allowable values:
       </p>
 
       <emu-table id="table-collator-options">
-        <emu-caption>Collator options settable through both extension keys and options properties</emu-caption>
+        <emu-caption>Collator options settable through extension keys, internal slots and options properties</emu-caption>
         <table class="real-table">
           <thead>
             <tr>
               <th>Key</th>
+              <th>Internal Slot</th>
               <th>Property</th>
               <th>Type</th>
               <th>Values</th>
             </tr>
           </thead>
           <tr>
             <td>kn</td>
+            <td>[[Numeric]]</td>
             <td>numeric</td>
             <td>"boolean"</td>
             <td></td>
           </tr>
           <tr>
             <td>kf</td>
+            <td>[[CaseFirst]]</td>
             <td>caseFirst</td>
```

Comment:
Editorial suggestion: now that this is just a property name and not also an internal slot, maybe it should be written as `"caseFirst"` rather than caseFirst.